users/jdelkins/jdelkins.c

```diff
@@ -0,0 +1,272 @@
+            do_mac_key(LCTL(KC_INS), RCMD(KC_C), record);
+            break;
+
+        case KB_PASTE:
+            do_mac_key(LSFT(KC_INS), RCMD(KC_V), record);
+            break;
+
+        case MY_GUI:
+            do_mac_key(KC_LGUI, KC_LOPT, record);
+            break;
+
+        case MY_ALT:
+            do_mac_key(KC_LALT, KC_LCMD, record);
+            break;
+
+        case MY_RGUI:
+            do_mac_key(KC_RGUI, KC_ROPT, record);
+            break;
+
+        case MY_RALT:
+            do_mac_key(KC_RALT, KC_RCMD, record);
+            break;
+
+        case MY_CALC:
+            do_mac_key(KC_CALC, KC_F14, record);
+            break;
+
+        case KB_MAKE:
+            if (!get_mods()) {
+                if (!record->event.pressed)
+                    SEND_STRING("make " QMK_KEYBOARD ":" QMK_KEYMAP SS_TAP(X_ENTER));
+                return false;
+            }
+            break;
+
+        case KB_VRSN:
+            if (!get_mods()) {
+                if (!record->event.pressed) {
+                    if (user_config.system_mac) {
+                        SEND_STRING(QMK_KEYBOARD "/" QMK_KEYMAP " @ " QMK_VERSION " (mac mode)");
+                    } else {
+                        SEND_STRING(QMK_KEYBOARD "/" QMK_KEYMAP " @ " QMK_VERSION " (non-mac mode)");
+                    }
+                }
+                return false;
+            }
+            break;
+
+        case KB_BOOT:
+            if (!get_mods()) {
+                if (record->event.pressed) {
+                    boot_timer = timer_read32();
+                } else {
+                    if (timer_elapsed32(boot_timer) >= 500) {
+                        reset_keyboard();
+                    }
+                }
+                return false;
+            }
+            break;
+
+        case KB_FLSH:
+            if (!get_mods()) {
+                if (!record->event.pressed) {
+                    SEND_STRING("make " QMK_KEYBOARD ":" QMK_KEYMAP
+#if defined(__ARM__) || defined(__arm__)
+                            ":dfu-util"
+#elif defined(BOOTLOADER_DFU) || defined(BOOTLOADER_LUFA_DFU) || defined(BOOTLOADER_QMK_DFU)
+                            ":dfu "
+#elif defined(BOOTLOADER_HALFKAY)
+                            ":teensy "
+#elif defined(BOOTLOADER_CATERINA)
+                            ":avrdude "
+#else
+                            ":doflash "
+#endif
+                            SS_TAP(X_ENTER));
+                    reset_keyboard();
+                }
+                return false;
+            }
+            break;
+
+#ifdef DO_SECRETS
+        case KC_SECRET_1 ... KC_SECRET_5: // Secrets!  Externally defined strings, not stored in repo
+            if (!record->event.pressed) {
+                clear_oneshot_layer_state(ONESHOT_OTHER_KEY_PRESSED);
+                send_secret_string(keycode - KC_SECRET_1);
+            }
+            return false;
+            break;
+#endif
+    }
+
+    return process_record_keymap(keycode, record);
```

Comment:
This may be better at the begining of the function, as: 
```c
    if (!process_record_keymap(keycode, record)) { return false; }
```

This way, if the keymap function returns false, it stops the processing of further keycodes, which is more in line with the rest.

Reply:
Done, thanks